File: snippets/visualbasic/VS_Snippets_CLR/Environment.MachineName/VB/machinename.vb
'<snippet1>
' Sample for the Environment.MachineName property
Class Sample
   Public Shared Sub Main()
      Console.WriteLine()
      '  <-- Keep this information secure! -->
      Console.WriteLine("MachineName: {0}", Environment.MachineName)
   End Sub
End Class
'
'This example produces the following results:
'
'MachineName: MyDesktop
'
'</snippet1>

File: snippets/cpp/VS_Snippets_CLR/Environment.MachineName/CPP/machinename.cpp

//<snippet1>
// Sample for the Environment::MachineName property
using namespace System;
int main()
{
   Console::WriteLine();
   
   //  <-- Keep this information secure! -->
   Console::WriteLine( "MachineName: {0}", Environment::MachineName );
}

/*
This example produces the following results:

MachineName: MyDesktop
*/
//</snippet1>
